bin/katello-certs-check

```diff
@@ -74,6 +74,16 @@ function check-expiration () {
     fi
 }
 
+function check-cert-ca-flag () {
+    printf "Checking if server cert has CA:TRUE flag"
+    openssl x509 -in $CERT_FILE -noout -text | grep -o CA:TRUE
```

Comment:
@ekohl good point, didnt know that flag existed, will change up and resubmit. I agree quiet should be used.